quickwit/quickwit-serve/src/lib.rs

```diff
@@ -1205,7 +1210,15 @@ async fn node_readiness_reporting_task(
     ingester_opt: Option<impl IngesterService>,
     grpc_readiness_signal_rx: oneshot::Receiver<()>,
     rest_readiness_signal_rx: oneshot::Receiver<()>,
+    mut health_reporter: HealthReporter,
 ) {
+    let mut node_ready = false;
+    cluster.set_self_node_readiness(node_ready).await;
+    // Set the initial health status to not serving with "" meaning all services.
```

Comment:
is that a convention we choose, or a standard? tonic_health doesn't seem to say anything about empty service names being special

Reply:
This is standard:
> The server should use an empty string as the key for server's overall health status

https://github.com/grpc/grpc/blob/master/doc/health-checking.md

Reply:
(I updated the comment)